src/main/java/com/jmb/springfactory/service/worklog/WorkLogService.java

```diff
@@ -0,0 +1,15 @@
+package com.jmb.springfactory.service.worklog;
+
+import java.util.List;
+
+import com.jmb.springfactory.exceptions.NotFoundException;
+import com.jmb.springfactory.exceptions.ServiceLayerException;
+import com.jmb.springfactory.model.dto.WorkLogDto;
+import com.jmb.springfactory.service.GenericService;
+
+public interface WorkLogService extends GenericService<WorkLogDto, Integer> {
+
+  public WorkLogDto save(Integer idTask, WorkLogDto workLog) throws NotFoundException, ServiceLayerException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)